src/snowflake/cli/_plugins/stage/manager.py

```diff
@@ -343,7 +355,7 @@ def execute(
             )
 
         for file_path in sorted_file_path_list:
-            file_stage_path = stage_path_parts.add_stage_prefix(file_path)
+            file_stage_path = self.get_standard_stage_prefix(file_path)
```

Comment:
Is this needed if all file paths have stage prefix?

Reply:
Yes, since some of them comes without the @